frontend/src/components/Header/index.tsx

```diff
@@ -21,6 +22,11 @@ const Header: FC = () => {
       <MainWrapper>
         <HomepageLink />
         <Right>
+          <a href={DOCS_HOME_PAGE} target="_blank">
```

Comment:
Add `noopener` for security!

```suggestion
          <a href={DOCS_HOME_PAGE} target="_blank" rel="noopener">
```